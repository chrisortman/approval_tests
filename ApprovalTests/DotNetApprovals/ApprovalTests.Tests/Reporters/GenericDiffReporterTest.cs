using ApprovalTests.Reporters;
using NUnit.Framework;

namespace ApprovalTests.Tests.Reporters
{
	[TestFixture]
	public class GenericDiffReporterTest
	{
		[Test]
		public void TestProgramsExist()
		{
			Assert.IsFalse(new GenericDiffReporter("this_should_never_exist", "").IsWorkingInThisEnvironment());
		}
	}
}