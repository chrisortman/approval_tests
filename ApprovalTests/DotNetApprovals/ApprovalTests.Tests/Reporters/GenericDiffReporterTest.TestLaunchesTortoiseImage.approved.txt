C:\Program Files (x86)\TortoiseSVN\bin\TortoiseIDiff.exe /left:"../../b.png" /right:"../../a.png"